tests/components/sensor/test_openhardwaremonitor.py

```diff
@@ -0,0 +1,52 @@
+"""The tests for the Open Hardware Monitor platform."""
+import unittest
+import requests_mock
+from homeassistant.components.sensor import openhardwaremonitor
+from homeassistant.setup import setup_component
+
+from tests.common import load_fixture, get_test_home_assistant
+
+
+class TestOpenHardwareMonitorSetup(unittest.TestCase):
+    """Test the Open Hardware Monitor platform."""
+
+    def add_entities(self, new_entities, update_before_add=False):
+        """Mock add entities."""
+        if update_before_add:
+            for entity in new_entities:
+                entity.update()
+
+        for entity in new_entities:
+            self.entities.append(entity)
+
+    def setUp(self):
+        """Initialize values for this testcase class."""
+        self.hass = get_test_home_assistant()
+        self.config = {
+            'host': 'localhost',
+            'port': 8085
+        }
+        self.entities = []
+
+    def tearDown(self):  # pylint: disable=invalid-name
+        """Stop everything that was started."""
+        self.hass.stop()
+
+    def test_setup_with_config(self):
+        """Test the platform setup with configuration."""
+        self.assertTrue(
```

Comment:
This test is just connecting to something that doesn't exist, and thus makes not much sense ? Also, this is already tested in the other method?